src/OSPSuite.R/Services/DataImporter.cs

```diff
@@ -191,10 +191,17 @@ private ColumnInfo createErrorColumn(ColumnInfo timeColumn, ColumnInfo concentra
          return errorInfo;
       }
 
-      public ImporterConfiguration ConfigurationFromData(string dataPath, IReadOnlyList<ColumnInfo> columnInfos, IReadOnlyList<MetaDataCategory> metaDataCategories)
+      public ImporterConfiguration ConfigurationFromData(string dataPath, IReadOnlyList<ColumnInfo> columnInfos, IReadOnlyList<MetaDataCategory> metaDataCategories, string sheetName)
```

Comment:
I don't understand WHY we have exactly the same code between R services and Presentation Service.s Shouldn't that be a common service?🤔

Reply:
You are right... Already created an abstract base class 🤔